src/client.js

```diff
@@ -340,7 +340,7 @@ Client.prototype.names = function(data) {
 	}
 
 	client.emit("names", {
-		chan: target.chan.id,
```

Comment:
What do you mean by `consistency with the remaining of the code`? Changing `chan` to `id` allows the client side to use the same function to render user list without thinking about which variable to use.